python/doc/examples/reliability_sensitivity/reliability/plot_post_analytical_importance_sampling.py

```diff
@@ -1,6 +1,6 @@
 """
-Post analytical importance sampling
-===================================
+Use the Post analytical importance sampling algorithm
```

Comment:
```suggestion
Use the post-analytical importance sampling algorithm
```
Not sure about this.